__tests__/app.test.js

```diff
@@ -142,4 +142,56 @@ describe("GET /api/reviews/:review_id", () => {
                 expect(body.msg).toBe("Not Found.")
             })
     })
+})
+
+describe("6. GET /api/reviews/:review_id/comments", () => {
+    test("Responds with an array of comments for the given review_id, each with all comment's properties", () => {
+        return request(app)
+            .get('/api/reviews/2/comments')
+            .then(( {body} ) => {
+                const {comments} = body;
+                expect(comments).toBeInstanceOf(Array);
+                expect(comments).toHaveLength(3);
+                comments.forEach((comment) => {
+                    expect(comment).toEqual(
+                        expect.objectContaining({
+                            comment_id: expect.any(Number),
+                            votes: expect.any(Number),
+                            created_at: expect.any(String),
+                            author: expect.any(String),
+                            body: expect.any(String),
+                            review_id: expect.any(Number)
+                        })
+                    )
+                })
+            })
+    })
+    test("Responds with a key of comments sorted by date (created_at) with the most recent first.", () => {
```

Comment:
We could probably combine this test block with the above as we are testing all of the default behaviours of this endpoint